docs/user/explanations/configuration.md

```diff
@@ -28,17 +28,19 @@ Epinio's configuration contains
   - Epinio API user name
   - Epinio API password
   - Epinio API certificate
-  
+
 The organization can be changed by running `epoinio target` with the
 name of the desired org as its single argument.
 
 User name and password are used by the client to authenticate itself
 when talking to Epinio's API server. The `epinio install` command
 saves the initial information to the configuration.
 
-The certificate information is set only when installing epinio for
-development. This uses an `omg.howdoi.website` domain with a
-self-signed certificate. `epinio install` saves the associated CA
+The certificate information is set only when installing epinio with
+`--tls-issuer=selfsigned-issuer`.  The installation uses a the wildcard domain
+`omg.howdoi.website` and the self-signed issuer by default.
```

Comment:
Hasn't this changed with #590 to always save the cert ?
Also `a the wild...`.

Reply:
I might not have understood that correctly. I think we always save the cert to the config, but this is about being able to specify some field on the cert?

I might need to look that up in code.

Reply:
Yes, I misunderstood and saving is explained in the next paragraph